connector.yaml

```
@@ -2,7 +2,26 @@ version: "1.0"
 specification:
   name: dynamodb
   summary: A DynamoDB source plugin for Conduit
-  description: A DynamoDB source plugin for Conduit, it scans the table at the beginning taking a snapshot, then starts listening to CDC events using DynamoDB streams.
+  description: |
+    ## Source
+    A source connector that pulls data from a DynamoDB table to downstream resources via Conduit.
+
+    The connector starts with a snapshot of the data currently existent in the table, sends these records to the 
```

This trailing space is causing all your troubles.
```suggestion
    The connector starts with a snapshot of the data currently existent in the table, sends these records to the
```

I also opened a PR for `conn-sdk-cli` that fixes this automatically: https://github.com/ConduitIO/conduit-connector-sdk/pull/293.

---

Thank you for the fix!